drape_frontend/base_renderer.hpp

```diff
@@ -67,7 +72,10 @@ class BaseRenderer : public MessageAcceptor
   TCompletionHandler m_renderingEnablingCompletionHandler;
   bool m_wasNotified;
 
-  void SetRenderingEnabled(bool const isEnabled, TCompletionHandler completionHandler);
+  atomic<bool> m_wasContextReset;
```

Comment:
Could you please explain why do you need atomic here and how you use synchronization related to it?